### DISCUSSION TITLE
How to disable (or enable) EcoMode via APRS message?

### DISCUSSION THREAD

**Discussion by MihaiRF** [richonguzman/LoRa_APRS_iGate#205] [General]
Hi all,
One of the latest releases feature the EcoMode function for getting the lowest possible power consumption for a remote digipeater. The change log says "APRS Message/Queries can start/stop this mode too". 
What would be the messageformat? Does it include any kind of "password" within the config?
Thank you!

**Comment by pdevlin13** [comment]
The message queries can be found in the WIKI here 
https://github.com/richonguzman/LoRa_APRS_iGate/wiki/104.-QUERIES

**Reply by MihaiRF** [reply]
Great info,
Thank you!
